src/import/FSHImporter.ts

```diff
@@ -700,6 +772,14 @@ export class FSHImporter extends FSHVisitor {
     return concept;
   }
 
+  visitSource(ctx: pc.SourceContext): string {
+    return this.aliasAwareValue(ctx.SEQUENCE());
```

Comment:
Should there be a test in `FSHImporter.Mapping.test.ts` checking that Alias's are supported on `Source`? Or is that covered by other Alias-related tests elsewhere?

Reply:
Yup, good point!